modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -85,7 +85,7 @@ def upsale; end
 
   def select_provider
     @object = Storages::Storage.new(provider_type: @provider_type)
-    service_result = ::Storages::Storages::SetProviderFieldsAttributesService
+    service_result = ::Storages::Storages::SetAttributesService
```

Comment:
🟢 Q: Is this no longer needed for Nextcloud default `automatically_managed = true` setting? FWIW, testing went Ok just seeking clarification.

Reply:
Nextcloud setup do not need to set the provider fields in the `select_provider` action (which is the first one executed when the multistep form is loaded, i.e. "empty general info"). The provider field defaults are inserted into the model before entering the last form step, with the method called `prepare_storage_for_automatic_management_form` in the `oauth_clients_controller`.

Reply:
It was needed before, as OneDrive setup had the checkbox for AMPF in the general info step.